From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 27 May 2017 12:57:22 +0200
Subject: mkinitrd: Add missing fscrypto module

---
 libvirt-sandbox/libvirt-sandbox-builder-machine.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libvirt-sandbox/libvirt-sandbox-builder-machine.c b/libvirt-sandbox/libvirt-sandbox-builder-machine.c
index bdec490..7204f71 100644
--- a/libvirt-sandbox/libvirt-sandbox-builder-machine.c
+++ b/libvirt-sandbox/libvirt-sandbox-builder-machine.c
@@ -186,6 +186,7 @@ static gchar *gvir_sandbox_builder_machine_mkinitrd(GVirSandboxConfig *config,
 
     /* In case ext4 is built as a module, include it and its deps
      * for the root mount */
+    gvir_sandbox_config_initrd_add_module(initrd, "fscrypto.ko");
     gvir_sandbox_config_initrd_add_module(initrd, "mbcache.ko");
     gvir_sandbox_config_initrd_add_module(initrd, "jbd2.ko");
     gvir_sandbox_config_initrd_add_module(initrd, "crc16.ko");
